tests-ng/test_umask.py

```diff
@@ -9,6 +9,7 @@ def test_umask_file(parse_file: ParseFile):
 
 
 @pytest.mark.root
+@pytest.mark.feature("not cis", reason="CIS needs umask 0027")
```

Comment:
this test asserts that the umask is 0027, I don't understand the provided reason. is there a typo?